doc/attr.xml

```diff
@@ -1352,3 +1352,38 @@ gap> HamiltonianPath(g);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="VertexConnectivity">
+<ManSection>
+  <Attr Name="VertexConnectivity" Arg="digraph"/>
+  <Returns>An non-negative integer.</Returns>
+  <Description>
+    For a digraph <A>digraph</A> with set of vertices <C>V</C>, the attribute
+    <C>VertexConnectivity(<A>digraph</A>)</C> returns the least cardinality
+    <C>|S|</C> of a subset <C>S</C> of <C>V</C> such that their removal either
+    disconnects <A>digraph</A>, makes it trivial, or makes it have no vertices.
+    The alrorithm makes <C>O(n - d - 1 + d * (d - 1) / 2)</C> calls to a
+    max-flow algorithm wich itself has complexity <C>O(n ^ 2 * e)</C>, where
+    <C>n</C> is the number of vertices of <A>digraph</A>, and <C>e, d</C> are
+    the number of edges and the degree (respectively) of the symmetric closure
```

Comment:
What is the degree of a graph?

Reply:
The degree of a digraph is the minimum of all the in-degrees and out-degrees of all vertices in the graph. Should I have specified this definition?

Reply:
You could therefore replace "degree" by "the minimum out-degree" to avoid using this new notion.